openml/tasks/task.py

```diff
@@ -7,38 +7,58 @@
 import openml._api_calls
 from ..utils import _create_cache_directory_for_id
 
-
 class OpenMLTask(object):
-    def __init__(self, task_id, task_type_id, task_type, data_set_id,
-                 target_name, estimation_procedure_type, data_splits_url,
-                 estimation_parameters, evaluation_measure, cost_matrix,
-                 class_labels=None):
+    def __init__(self, task_id, task_type_id, task_type, data_set_id, estimation_procedure_type,
+                 estimation_parameters, evaluation_measure):
         self.task_id = int(task_id)
         self.task_type_id = int(task_type_id)
         self.task_type = task_type
         self.dataset_id = int(data_set_id)
```

Comment:
can we call this "source_data", in co-ordinance with the task descriptions (hidden in the XML)